server/api/swagger/swagger.yaml

```diff
@@ -4321,11 +4321,52 @@ paths:
                 type: number
               size:
                 type: number
+              token:
+                type: string
             required:
               - symbol
               - price
               - side
               - size
+              - token
+      responses:
+        200:
+          description: Success
+          schema:
+            $ref: "#/definitions/ObjectResponse"
+        default:
+          description: Error
+          schema:
+            $ref: "#/definitions/MessageResponse"
+      security:
+        - Bearer: []
+      x-security-scopes:
+        - user
+  /broker/quote:
```

Comment:
@fetok12 this endpoint should does not need to have Bearer. It should work publicly as mentioned in the requirements